Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -2673,14 +2673,10 @@ internal void ApplySettings()
                 runPlayerUpdatesInEditMode = m_Settings.IsFeatureEnabled(InputFeatureNames.kRunPlayerUpdatesInEditMode);
                 #endif
 
-                #if UNITY_EDITOR_WIN || UNITY_STANDALONE_WIN
-                if (m_Settings.IsFeatureEnabled(InputFeatureNames.kUseWindowsGamingInputBackend))
```

Comment:
Is this being removed in this PR - I've seen it discussed elsewhere but seems outside scope of analytics

Reply:
That was removed with https://github.com/Unity-Technologies/InputSystem/pull/1955, is this comment made on a partial diff view? It seems like it is